snappi_ixnetwork/snappi_api.py

```diff
@@ -669,6 +637,225 @@ def update_flows(self, payload):
         except Exception as err:
             raise SnappiIxnException(err)
         return self._request_detail()
+    
+    def _get_max_convergence(self, rows):
+        # We are extracting max value for multiple destination
+        rows_len = len(rows)
+        if rows_len > 1:
+            for (
+                _,
+                internal_name,
+                external_type,
+            ) in self._CONVERGENCE:
+                if external_type not in ["float", "int"]:
+                    continue
+                max_value = rows[0][internal_name]
+                for idx in range(1, rows_len):
+                    tmp_value = rows[idx][internal_name]
+                    if max_value < tmp_value:
+                        max_value = tmp_value
+                rows[0][internal_name] = max_value
+        return rows[0]
+    
+    def _result(self, request):
+        # get convergence result
+        flow_names = request.get("flow_names")
+        if not isinstance(flow_names, list):
+            msg = "Invalid format of flow_names passed {},\
+                                                expected list".format(
+                flow_names
+            )
+            raise Exception(msg)
+        if flow_names is None or len(flow_names) == 0:
+            flow_names = [flow.name for flow in self.snappi_config.flows]
+        flow_rows = self._get_flow_rows(flow_names)
+        traffic_stat = self._assistant.StatViewAssistant(
+            "Traffic Item Statistics"
+        )
+        traffic_index = {}
+        drill_down_option = "Drill down per Dest Endpoint"
+        for index, row in enumerate(
+            self._get_traffic_rows(traffic_stat, drill_down_option)
+        ):
+            traffic_index[row["Traffic Item"]] = index
+        response = []
+        for flow_name in flow_names:
+            traffic_stat = self._assistant.StatViewAssistant(
+                "Traffic Item Statistics"
+            )
+            convergence = {"name": flow_name}
+            if flow_name not in traffic_index.keys():
+                raise Exception("Somehow flow %s is missing" % flow_name)
+
+            drill_down_options = traffic_stat.DrillDownOptions()
+            drilldown_index = drill_down_options.index(drill_down_option)
+            drill_down = traffic_stat.Drilldown(
+                drilldown_index,
+                drill_down_option,
+                traffic_stat.TargetRowFilters()[traffic_index[flow_name]],
+            )
+            drill_down_result = self._get_max_convergence(drill_down.Rows)
```

Comment:
expecting drill down happen before that. It will better to check Row has some length.

Reply:
Done.